src/index.js

```diff
@@ -47,6 +78,15 @@ class TracePlugin {
       'post:setup': this.postSetup.bind(this),
       'pre:report': this.preReport.bind(this)
     };
+    if (this.config.autoHttp.enabled) {
+      this.autoData = {
+        timeline: new Perf({ timestamp: true }),
```

Comment:
Can't an invocation share the same timeline instance? That's what I'm doing on the Python side.

Reply:
Result: one timeline for each plugin instance, clear timeline after each invocation

Reply:
Node plugin architecture is such that a new plugin instance is created per invocation. So each invocation will have it's own timeline.